webpack/ForemanRhCloudHelpers.js

```diff
@@ -11,3 +11,7 @@ export const isNotRhelHost = ({ hostDetails }) =>
     // eslint-disable-next-line camelcase
     hostDetails?.operatingsystem_name
   );
+
+export const isNonRhelAndNoLocalAdvisor = ({ hostDetails }) =>
```

Comment:
```suggestion
export const vulnrabilityEnabled = ({ hostDetails }) =>
```
I think it would be a bit more indicative